### ISSUE TITLE
Type Declaration Issue with Oruga v0.8.0 '@oruga-ui/oruga-next'

### ISSUE THREAD

**Issue by messenjer** [oruga-ui/oruga#678]
### Overview of the problem

**Oruga** version: 0.8.0
**Vuejs** version: 3.3.10
**OS/Browser**: Mac Chrome

### Description

When attempting to use the '@oruga-ui/oruga-next' module in a project, the TypeScript compiler raises the following error:

```
Could not find a declaration file for module '@oruga-ui/oruga-next'.
'/home/messenjer/oruga-type-error/node_modules/@oruga-ui/oruga-next/dist/esm/index.mjs' implicitly has an 'any' type.
There are types at '/home/messenjer/oruga-type-error/node_modules/@oruga-ui/oruga-next/dist/types/index.d.ts', but this result could not be resolved when respecting package.json "exports".
The '@oruga-ui/oruga-next' library may need to update its package.json or typings.ts(7016).
```

### Steps to reproduce

Install a fresh new vue application

```
npm create vue@latest
npm i  @oruga-ui/oruga-next
```

Repro : https://stackblitz.com/~/github.com/messenjer/oruga-type-error 

<img width="1056" alt="Screenshot 2023-12-08 at 11 31 56" src="https://github.com/oruga-ui/oruga/assets/80677/ac9090d9-5d69-4f38-8f78-a74e6d6bd1c3">

### Expected behavior

Developers expect to use the '@oruga-ui/oruga-next' module seamlessly with TypeScript

### Actual behavior

Type Declaration Issue

**Comment by mlmoravek** [comment]
Hey @messenjer, thanks for this issue. 
I'm kind of new to this topic. 
Do you think this little fix #681 might fix this?

**Comment by mlmoravek** [comment]
Good 👍 Will do the release tomorrow i think :)
